dataplug-facebook/app/com/hubofallthings/dataplugFacebook/apiInterfaces/FacebookEventInterface.scala

```diff
@@ -21,7 +21,7 @@ import com.hubofallthings.dataplug.services.UserService
 import com.hubofallthings.dataplug.utils.{ AuthenticatedHatClient, FutureTransformations, Mailer }
 import com.hubofallthings.dataplugFacebook.models.FacebookEvent
 import com.mohiva.play.silhouette.api.repositories.AuthInfoRepository
-import com.mohiva.play.silhouette.impl.providers.oauth2.FacebookProvider
+import com.hubofallthings.dataplugFacebook.apiInterfaces.authProviders._
```

Comment:
I notice I have the ._ when not needed. It should make explicit of the provider, It's one class only. Just a small visual thing

Reply:
there is quite a few changes likes that across different data plugs, they have happened in a fairly recent commits